pkg/controller/machine/machine_controller.go

```diff
@@ -18,18 +18,25 @@ package machine
 
 import (
 	"context"
+	"fmt"
 	"path"
 	"sync"
+	"time"
 
+	"github.com/go-log/log/info"
+	kubedrain "github.com/openshift/kubernetes-drain"
 	"github.com/pkg/errors"
 	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
 	"k8s.io/klog"
 	"k8s.io/utils/pointer"
 	clusterv1 "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha2"
 	capierrors "sigs.k8s.io/cluster-api/pkg/controller/error"
+	controllerError "sigs.k8s.io/cluster-api/pkg/controller/error"
```

Comment:
Package imports should be all lowercase

Reply:
FYI this is not net new here. I stumbled across this alias already in use in the repo elsewhere. I'll file an issue to fix it.

Reply:
I missed it, apologies, you can assign it to me

Reply:
Filed #1097